python/doc/examples/reliability_sensitivity/reliability/plot_nais.py

```diff
@@ -235,5 +222,40 @@ def fourBranch(x):
     dr.setLineWidth(3)
     graph.add(dr)
 
-graph.setColors(col)
-view = View(graph)
+graph.setLegendPosition("lower left")
```

Comment:
```suggestion
graph.setColors(col)
graph.setLegendPosition("lower left")
```

Sorry about that change, I was wrong. The drawable `dr` is obtained from the graph produced by `draw`, which means its color was explicitly set, so the default colors do not apply.